### PR TITLE
Update Italian translations

### PR DESCRIPTION
**Pull request by abonte**

### CODE REVIEW

File: CliClient/locales/it_IT.po [hunk 1]
```diff
@@ -657,17 +657,16 @@ msgid "Cancelling... Please wait."
...
 "<tag-command> can be \"add\", \"remove\", \"list\", or \"notetags\" to "
 "assign or remove [tag] from [note], to list notes associated with [tag], or "
 "to list tags associated with [note]. The command `tag list` can be used to "
 "list all the tags (use -l for long option)."
 msgstr ""
-"<tag-command> può essere \"add\", \"remove\" or \"list\" per assegnare o "
-"rimuovere [tag] da [note], o per mostrare le note associate a [tag]. Il "
-"comando `tag list` può essere usato per mostrare tutte le etichette (usa -l "
-"per l’opzione lunga)."
+"<tag-command> può essere \"add\", \"remove”, \"list\", or “notetags” per "
```
Comment (tessus) [inline CliClient/locales/it_IT.po]: The quotes around `notetags` and at the end of `remove` are typographic ones. For consistency, please use straight double (standard) quotes.

Comment (abonte) [reply CliClient/locales/it_IT.po]: Fixed!

Comment (tessus) [reply CliClient/locales/it_IT.po]: Awesome. Thx.

**Comment by tessus** [comment]
Thanks for the update!

**Pull request merged**